npm/asciidoctor-revealjs

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+'use strict'
+
+process.title = 'asciidoctor-revealjs'
+
+const { Options, Invoker } = require('@asciidoctor/cli')
```

Comment:
Is it working? I'm surprise because you are using `require('@asciidoctor/cli')` but `@asciidoctor/cli` is not a direct `dependencies` in the root `package.json`?